app/main.py

```diff
@@ -1,3 +1,2 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    return phrase.lower().count(letter.lower())
```

Comment:
The return statement is incorrect for the intended functionality. The `count` method already returns an integer, so dividing it by anything (or in this case, not dividing but just returning) is unnecessary. Ensure that the function returns the count directly as an integer.